README.md

```diff
@@ -61,6 +61,17 @@ or instantiate the client with the valid configuration:
 UNLEASH = Unleash::Client.new(url: 'http://unleash.herokuapp.com/api', app_name: 'my_ruby_app', custom_http_headers: {'Authorization': '<API token>'})
 ```
 
+## Dynamic custom HTTP headers
+If you need custom http headers that change during the lifetime of the client, a the `custom_http_headers` can be given as a `Proc`.
+
+```ruby
+Unleash.configure do |config|
+  config.app_name            = 'my_ruby_app'
+  config.url                 = 'http://unleash.herokuapp.com/api'
```

Comment:
```suggestion
  config.url                 = 'https://unleash.herokuapp.com/api'
```

Reply:
This was copied from the example above, should we change both places?

Reply:
If you don't mind! It's a very small thing but its nice to keep this kind of stuff updated when we are touching the code anyway